.github/workflows/create-release.yml

```
@@ -0,0 +1,368 @@
+        with:
+          fetch-depth: 0
+          token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Configure Git
+        run: |
+          git config user.name "${{ env.GH_USER_NAME }}"
+          git config user.email "${{ env.GH_USER_EMAIL }}"
+
+      - name: Set version variables
+        id: set_vars
+        run: |
```

actionlint (1.7.10)

shellcheck reported issue in this script: SC2086:info:14:28: Double quote to prevent globbing and word splitting

(shellcheck)